app/spaces/[spaceid]/setting/apps/page.tsx

```
@@ -0,0 +1,209 @@
+        filters: {
+          where: {
+            isInstallable: {
+              equalTo: '1',
+            },
+          },
+        },
+      });
+
+      if (
+        response &&
+        response.data &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)